exercises/leap/instructions.md

```diff
@@ -0,0 +1,3 @@
+# Instructions
+
+Your task is to determine if any given year is a leap year.
```

Comment:
I like that this is now more on track with the "modern" concept exercise style files